core/dbt/events/base_types.py

```diff
@@ -37,6 +36,20 @@ def level_tag(self) -> str:
         return "error"
 
 
+class Node():
```

Comment:
using `@dataclass` would be a bit cleaner here than using an `__init__()` method.